istio-operator/istio-operator.spec

```diff
@@ -22,7 +22,14 @@
 %global repo            istio-operator
 
 # are we building community or product rpms
-%global community_build  true
+%global community_build  0
+
+%if 0%{?community_build}
```

Comment:
is there a reason we can't use true/false here?

Reply:
My understanding is that these evaluate 0/1 and if you use strings, you have to match them that way, which prevents the use of defaulting.  TBH, I'm no expert, but based on my limited search, this seemed like the easiest approach.  I'm happy to change it if there's a better approach.

Reply:
Why don't we get rid of community_build and just use build_type as a global that either has teh value "maistra" or "servicemesh" defined as:

%global build_type maistra
or 
%global build_type servicemesh

?